requirements.txt

```diff
@@ -1,4 +1,4 @@
-fonttools==3.0
+git+https://github.com/behdad/fonttools.git@7f5d5454fc1332815a42ba8b9fb666981680bce4#egg=fonttools
```

Comment:
good idea to freeze to a specific commit in time, instead of pulling it from HEAD.
I'd do it for all other requirements as well, so that one knows at any point in time the stable, supported state of all the dependencies.